src/libxively/control_topic/xi_control_message.h

```diff
@@ -54,6 +54,8 @@ typedef struct xi_control_message_file_desc_ext_s
 
     uint8_t* fingerprint;
     uint16_t fingerprint_len;
+
+    uint8_t is_processed;
```

Comment:
This is a protocol descriptor file. Adding a local client state specific field to a SFT protocol message is a mixing of different areas: SFT protocol and the client's status of package download. Even if this info will be needed in the final solution it'd be better to store outside the client side protocol description.